string name
float64 elapsed_time
float64 init_angle
ActionData panels_action_data
ActionData sunled_action_data
